pallets/parachain-staking/src/migrations.rs

```diff
@@ -86,6 +86,11 @@ where
 
 		assert!(!count_freezes.is_zero(), "Staking: There are still no freezes.");
 
+		pallet_balances::Freezes::<T>::iter().try_for_each(|(key, details)| {
```

Comment:
Please Ignore. 
Debugging Artifact. 
removed in: [4f1d80](https://github.com/KILTprotocol/kilt-node/pull/527/commits/4f1d80cf1f04ea03a53f94d5ff65f3e1a379a93c)